### PR TITLE
feat(//docker): New base container to let master build in container

### PR DESCRIPTION
**Pull request by narendasan**
# Description

Change the base container for the Dockerfile most users will use. Move NGC container to Dockerfile.ngc

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -44,20 +44,20 @@ RUN rm -rf /opt/pytorch/torch_tensorrt
 # copy source repo
 COPY . /workspace/torch_tensorrt
 COPY --from=torch-tensorrt-builder  /workspace/torch_tensorrt/src/py/dist/ .
-RUN conda init bash
+#RUN conda init bash
```
Comment (andi4191) [inline docker/Dockerfile:47]: Can we get rid of commented code?

File: docker/Dockerfile.ngc [hunk 2]
```diff
@@ -0,0 +1,63 @@
+ARG BASE=21.10
```
Comment (andi4191) [inline docker/Dockerfile.ngc:1]: We can bump this version to 22.04 as well.

File: docker/README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.
 
 ```
 # Build:
-DOCKER_BUILDKIT=1 docker build --build-arg BASE={PyTorch Base Container Version} -f docker/Dockerfile -t torch_tensorrt1.0:latest .
+DOCKER_BUILDKIT=1 docker build --build-arg BASE={TensorRT Base Container Version} -f docker/Dockerfile -t torch_tensorrt1.0:latest .
```
Comment (andi4191) [inline docker/README.md:13]: Should we use NGC CUDA container instead?
CI will be based on NGC CUDA container.

This way we have:
NGC branch from NGC PyTorch container
Master branch from TensorRT container
CI from NGC CUDA container

**Review by andi4191 — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 3 above)
Comment (narendasan) [reply docker/README.md:13]: We need tensorrt for external use since there's no curl able public trt package

**Review by andi4191 — APPROVED** [review]
LGTM

**Pull request merged**